### ISSUE TITLE
Units 7B and 13L missing from 2017 harvest data

### ISSUE THREAD

**Issue by 12382** [Huh/PopR_SDGFP#188]
I was creating a harvest summary of PopR data of each DAU and then adding them together to create a statewide summary.  When I compared this to what PopR produces for 2017 I noticed a discrepancy in the values.  It seems that there are 45 female and 17 male whitetails harvested and 3 male mule deer that don't have a DAU, or even a Unit associated with them.  Corey Huxoll believes that these deer are associated with Units 7B and 13L.  Cory says the sql tables have units specified for everything.  Is it possible that PopR just doesn't have these units specified geographically?

**Comment by Huh** [comment]
@paul-lukacs

Referencing Paul to make sure he gets this message.  That said, I am not convinced that the estimates will scale perfectly as you described.  Why not just fit the statewide estimate?

As to the units issue, we will look into it to see how they relate.

**Comment by paul-lukacs** [comment]
Yes  - PopR  harvest analyses do not know about 7B and 13L.  Perhaps there is a protocol we could set up for sending us regulation updates when they occur or once a year?

I assume you are referring to the harvest summaries in popr.cfc.umt.edu/SDGFP.  Once we get 7B and 13L in there, it all should work.

If you were referring to the harvest survey analysis, when I tried to get 2017 data for the harvest survey analysis, there wasn’t any available.  That is correct given my last update from Corey.

**Comment by 12382** [comment]
You are correct, the Statewide estimate is what should ultimately be used.  I was using this addition system to make sure the DAU estimates were reasonable and try to find discrepancies or outliers such as these.  Let me know when the units are added and I will run 2017 again.

Thanks
